guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -42,22 +42,68 @@ ifdef::katello[]
 ----
 # {package-manager} config-manager --set-enabled powertools
 ----
+
++
+. Enable the Katello module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable katello:el8
+----
 endif::[]
-ifdef::foreman-el,katello[]
+
+=== In case module enablement fails
+
+If the module fails to enable, it can mean an incorrect module is enabled. List the enabled modules to find out which ones:
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
+
+==== Ruby
+
+If the Ruby 2.5 module has already been enabled, you need to perform a module reset:
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module reset ruby
+----
+
+==== PostgreSQL
+
+If the PostgreSQL 10 module has already been enabled, you need to perform a module reset:
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module reset postgresql
+----
+
+When a database was previously created using PostgreSQL 10, an upgrade must be performed:
+
+. Enable the module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+ifdef::katello[]
+# dnf module enable katello:el8
+endif::[]
+ifndef::katello[]
+# dnf module enable foreman:el8
+endif::[]
+----
+
 +
-. Enable the PostgreSQL 12 module:
+. Install the PostgreSQL upgrade package:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-manager} module enable postgresql:12
+# dnf install postgresql-upgrade
 ----
+
 +
-. If the PostgreSQL 10 module has already been enabled, a module reset will need to be performed.
+. Perform the upgrade:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-manager} module reset postgresql
-# {package-manager} module enable postgresql:12
+# postgresql-setup --upgrade
 ----
```

Comment:
This whole thing would be another description list item with a nested numbered list. Make sure it's actually nested ;)